packages/next/src/index.ts

```diff
@@ -10,19 +10,20 @@ import type { LogtoNextConfig } from './types';
 
 export { ReservedScope, UserScope } from '@logto/node';
 
-export type { LogtoContext } from '@logto/node';
+export type { LogtoContext, InteractionMode } from '@logto/node';
 
 export default class LogtoClient {
   private navigateUrl?: string;
   private storage?: NextStorage;
   constructor(private readonly config: LogtoNextConfig) {}
 
   handleSignIn = (
-    redirectUri = `${this.config.baseUrl}/api/logto/sign-in-callback`
+    redirectUri = `${this.config.baseUrl}/api/logto/sign-in-callback`,
+    interactionMode?: InteractionMode
   ): NextApiHandler =>
     withIronSessionApiRoute(async (request, response) => {
       const nodeClient = this.createNodeClient(request);
-      await nodeClient.signIn(redirectUri);
+      await nodeClient.signIn(redirectUri, interactionMode);
```

Comment:
The `handleAuthRoutes` need to update? Like what you did in express: add a "sign-up" route.

Reply:
@wangsijie updated please check thx